006-securing-kafka-connect-rest-endpoint.md

```diff
@@ -0,0 +1,197 @@
+# Securing the Kafka Connect REST API endpoint
```

Comment:
I do not think securing is the right word given this covers only TLS encryption. It should be clear from the name that this is encryption only.